openmetadata-ui/src/main/resources/ui/src/components/Explore/ExploreTree/ExploreTree.interface.ts

```diff
@@ -48,3 +50,276 @@ export type DatabaseFields =
+  DynamoDB: FormattedDatabaseServiceType.DynamoDB,
+  Exasol: FormattedDatabaseServiceType.Exasol,
+  Glue: FormattedDatabaseServiceType.Glue,
+  Greenplum: FormattedDatabaseServiceType.Greenplum,
+  Hive: FormattedDatabaseServiceType.Hive,
+  Iceberg: FormattedDatabaseServiceType.Iceberg,
+  Impala: FormattedDatabaseServiceType.Impala,
+  MariaDB: FormattedDatabaseServiceType.MariaDB,
+  MongoDB: FormattedDatabaseServiceType.MongoDB,
+  Mssql: FormattedDatabaseServiceType.Mssql,
+  Mysql: FormattedDatabaseServiceType.Mysql,
+  Oracle: FormattedDatabaseServiceType.Oracle,
+  PinotDB: FormattedDatabaseServiceType.PinotDB,
+  Postgres: FormattedDatabaseServiceType.Postgres,
+  Presto: FormattedDatabaseServiceType.Presto,
+  QueryLog: FormattedDatabaseServiceType.QueryLog,
+  Redshift: FormattedDatabaseServiceType.Redshift,
+  SAS: FormattedDatabaseServiceType.SAS,
+  SQLite: FormattedDatabaseServiceType.SQLite,
+  Salesforce: FormattedDatabaseServiceType.Salesforce,
+  SapERP: FormattedDatabaseServiceType.SapERP,
+  SapHana: FormattedDatabaseServiceType.SapHana,
+  SingleStore: FormattedDatabaseServiceType.SingleStore,
+  Snowflake: FormattedDatabaseServiceType.Snowflake,
+  Synapse: FormattedDatabaseServiceType.Synapse,
+  Teradata: FormattedDatabaseServiceType.Teradata,
+  Trino: FormattedDatabaseServiceType.Trino,
+  UnityCatalog: FormattedDatabaseServiceType.UnityCatalog,
+  Vertica: FormattedDatabaseServiceType.Vertica,
+
+  // Dashboard Services
+  CustomDashboard: FormattedDashboardServiceType.CustomDashboard,
+  DomoDashboard: FormattedDashboardServiceType.DomoDashboard,
+  Lightdash: FormattedDashboardServiceType.Lightdash,
+  Looker: FormattedDashboardServiceType.Looker,
+  Metabase: FormattedDashboardServiceType.Metabase,
+  MicroStrategy: FormattedDashboardServiceType.MicroStrategy,
+  Mode: FormattedDashboardServiceType.Mode,
+  PowerBI: FormattedDashboardServiceType.PowerBI,
+  PowerBIReportServer: FormattedDashboardServiceType.PowerBIReportServer,
+  QlikCloud: FormattedDashboardServiceType.QlikCloud,
+  QlikSense: FormattedDashboardServiceType.QlikSense,
+  QuickSight: FormattedDashboardServiceType.QuickSight,
+  Redash: FormattedDashboardServiceType.Redash,
+  Sigma: FormattedDashboardServiceType.Sigma,
+  Superset: FormattedDashboardServiceType.Superset,
+  Tableau: FormattedDashboardServiceType.Tableau,
+
+  // Messaging Services
+  CustomMessaging: FormattedMessagingServiceType.CustomMessaging,
+  Kafka: FormattedMessagingServiceType.Kafka,
+  Kinesis: FormattedMessagingServiceType.Kinesis,
+  Redpanda: FormattedMessagingServiceType.Redpanda,
+
+  // API Services
+  REST: FormattedAPIServiceType.REST,
+  Webhook: FormattedAPIServiceType.Webhook,
+} as const;
+
+// Type for the keys of the serviceTypeFormatters object
+export type ServiceType = keyof typeof serviceTypeFormatters;
+
+// Helper function to get formatted service type
+export const getFormattedServiceType = (serviceType: string): string => {
```

Comment:
An interface can't have a function move it out to a util. Also we need to check on Collate side we should be able to expand the serviceType on Collate